lib/diplomat/session.rb

```diff
@@ -3,7 +3,7 @@
 module Diplomat
   class Session < Diplomat::RestClient
 
-    @access_methods = [ :create, :destroy ]
+    @access_methods = [ :create, :destroy, :list ]
```

Comment:
Could you add renew to this list so it's accessible from `Diplomat::Session.list(id)`?